fedora/xccdf/system/accounts/physical.xml

```diff
@@ -125,39 +133,47 @@ the grub2 superuser account and password, please refer to
 
 <Rule id="bootloader_uefi_password" severity="medium">
 <title>Set the UEFI Boot Loader Password</title>
-<description>The UEFI grub2 boot loader should have a superuser account and password
+<description>The grub2 boot loader should have a superuser account and password
 protection enabled to protect boot-time settings.
 <br /><br />
-To do so, select a superuser account and password and add them into the
-<tt>/etc/grub.d/01_users</tt> configuration file.
+To do so, select a superuser account name and password and and modify the
+<tt>/etc/grub.d/01_users</tt> configuration file with the new account name.
 <br /><br />
 Since plaintext passwords are a security risk, generate a hash for the pasword
 by running the following command:
-<pre>$ grub2-mkpasswd-pbkdf2</pre>
-When prompted, enter the password that was selected and insert the returned
-password hash into the <tt>/etc/grub.d/01_users</tt> configuration file immediately
-after the superuser account.
-(Use the output from <tt>grub2-mkpasswd-pbkdf2</tt> as the value of
-<b>password-hash</b>):
-<pre>password_pbkdf2 <b>superusers-account</b> <b>password-hash</b></pre>
+<pre>$ grub2-setpassword</pre>
+When prompted, enter the password that was selected.
+<br /><br />
 NOTE: It is recommended not to use common administrator account names like root,
 admin, or administrator for the grub2 superuser account.
 <br /><br />
+Change the superuser to a different username (The default is 'root').
+<pre>$ sed -i s/root/bootuser/g /etc/grub.d/01_users</pre>
+<br /><br />
 To meet FISMA Moderate, the bootloader superuser account and password MUST
 differ from the root account and password.
 Once the superuser account and password have been added, update the
 <tt>grub.cfg</tt> file by running:
-<pre>grub2-mkconfig -o /boot/efi/EFI/fedora/grub.cfg</pre>
+<pre>grub2-mkconfig -o /boot/efi/EFI/redhat/grub.cfg</pre>
```

Comment:
Is this really ``EFI/redhat`` on a fedora box? I don't have access to one to double check.